src/main/java/step3/CarSimulator.java

```diff
@@ -0,0 +1,51 @@
+package step3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarSimulator {
+    private final int movementCount;
+    private final List<Car> cars;
+    private final List<CarPositions> history;
+
+    public CarSimulator(int carCount, int movementCount) {
+        this.movementCount = movementCount;
+        this.cars = new ArrayList<>();
+        this.history = new ArrayList<>();
+
+        for (int i = 0; i < carCount; i++) {
+            cars.add(new Car());
+        }
+    }
+
+    public CarPositions getPositions() {
+        List<String> positionsList = new ArrayList<>();
```

Comment:
```suggestion
        List<String> positions = new ArrayList<>();
```
이미 `positions`로 복수에 대한 표현을 하고 있기 때문에 이를 구현하고 있는 자료구조가 무엇인지(List)에 대해서는 표기하지 않아도 되겠네요 😄

Reply:
@jinyoungchoi95 
이 부분은 놓쳤네요 ㅎㅎ
감사합니다.